front_end/lib/ui/sign_in_page/sign_in_page_provider.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+import 'sign_in_page_bloc.dart';
+import 'sign_in_page_view.dart';
+
+class SignInPageProvider extends StatelessWidget {
```

Comment:
no need to use StatelessWidget.

![image](https://github.com/fcodelabs/training-ramp-up/assets/47570687/6f1feaaf-ecd4-4828-9a34-ad3db7c8609d)

refer this.